DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/AdminFieldsController.cs

```diff
@@ -518,5 +553,13 @@ private void ValidateBulkOptionsString(string? optionsString)
                 );
             }
         }
+
+        private bool VerifyAdminUserCanAccessCourse(int customisationId)

```

Comment:
Would it be worth pulling this into a custom attribute as it would be useful to have other endpoints with that check, looks like DI for the course service can be used for ServiceFilters so the call to service shouldn't be an issue :)

https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/filters?view=aspnetcore-3.1#dependency-injection

Reply:
Pulled out this controller method into a Service filter